<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QWidget" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="typesBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбор типов</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="type1Box">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>100</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип 1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="type2Box">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>100</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Принять</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QPushButton" name="informationButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Подробнее</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearStatisticsButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Очистить статистику</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="type3Box">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>100</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип 3</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="type4Box">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>100</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип 4</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
